onadata/settings/common.py

```diff
@@ -411,8 +408,8 @@ def configure_logging(logger, **kwargs):
 after_setup_logger.connect(configure_logging)
 
 GOOGLE_STEP2_URI = 'http://ona.io/gwelcome'
-GOOGLE_CLIENT_ID = '617113120802.onadata.apps.googleusercontent.com'
-GOOGLE_CLIENT_SECRET = '9reM29qpGFPyI8TBuB54Z4fk'
+GOOGLE_OAUTH2_CLIENT_ID = '617113120802.onadata.apps.googleusercontent.com'
+GOOGLE_OAUTH2_CLIENT_SECRET = '9reM29qpGFPyI8TBuB54Z4fk'
```

Comment:
We override this in deployment, I will check if empty strings do not affect tests.